src/pages/[platform]/build-a-backend/storage/storagepath/index.mdx

```diff
@@ -0,0 +1,103 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Using StoragePath',
+  description: "Learn more about constructing a StoragePath to use on Amplify Storage APIs",
+  platforms: [
+    'swift',
+    'android'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+<InlineFilter filters={["android", "swift"]}>
+You can now use `StoragePath` to explicitly access, upload, and download to any path in your S3 bucket.
+Instead of using access levels, which restrict you to the predefined `guest`, `private`, or `public` folder in S3, `StoragePath` gives you more flexibility and control to create, maintain, and provision the path and storage structure that best fits your needs.
```

Comment:
The definitions of public/protected/private are taken from the "Files Access Levels" page. I think it makes sense to duplicate some of that here since that page has a warning a the top with a link to this page.